code/modules/food_and_drinks/kitchen_machinery/kitchen_machine.dm

```diff
@@ -286,14 +286,18 @@
 					O.reagents.trans_to(temp_reagents, O.reagents.total_volume, no_react = TRUE) // Don't react with the abstract holder please
 				qdel(O)
 			source.reagents.clear_reagents()
-			for(var/e=1 to efficiency)		//upgraded machine? make additional servings and split the ingredient reagents among each serving equally.
+
+			var/reagents_per_serving = temp_reagents.total_volume/efficiency
```

Comment:
```suggestion
			var/reagents_per_serving = temp_reagents.total_volume / efficiency
```

Reply:
Probably best to put it in brackets too as this would be easier to read

Reply:
In this case it's just one operation, does it really need brackets?

Reply:
It's not needed for the operation but the brackets help draw someone's attention to the fact a calculation is happenning here